src/software/simulation/ssl_simulation_protocol/ssl_simulation_protocol_manager.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+class SslSimulationProtocolManager
+{
+  public:
+    /**
+     * Sets up the initial simulation with robots, ball state, etc
+     */
+    SslSimulationProtocolManager::setupInitialSimulationState()
+
+    /**
+     * Starts the simulation. If the simulator is already running, this
+     * function does nothing.
+     */
+    void startSimulation();
```

Comment:
Not exactly sure what you mean by that; to clarify what I was thinking for this is that this would start the protocol manager (including sending the commands via simulation protocol), but not grsim itself